src/widget/wtrackmenu.cpp

```diff
@@ -2170,6 +2210,27 @@ void WTrackMenu::slotClearHotCues() {
             &trackOperator);
 }
 
+void WTrackMenu::slotSortHotcuesByPosition(HotcueSortMode sortMode) {
+    QString progressLabelText;
+    switch (sortMode) {
+    case HotcueSortMode::RemoveOffsets:
+        progressLabelText = tr(
+                "Sorting hotcues of %n track(s) by position (remove offsets)",
+                "",
+                getTrackCount());
+        break;
+        //    case HotcueSortMode::KeepOffsets:
+        //        progressLabelText =
+        //                tr("Sorting hotcues of %n track(s) by position", "", getTrackCount());
+        //        break;
+        //    }
```

Comment:
context for any1 else, this is related to https://github.com/mixxxdj/mixxx/pull/13808#discussion_r1947831299